src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../Types';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  list: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ list = [] }) => (
```

Comment:
While your list inside the MovieList component, it's does not obey the rules of the grid.